_build/builder/BuildDocs.py

```diff
@@ -73,27 +88,19 @@ def Setup(self):
 
     def DoTask(self):
         tmpDir = join(self.buildSetup.tmpDir, "chm")
-        Prepare()
-        #warnings.simplefilter('ignore', DeprecationWarning)
-        sphinx.build_main([
-            None,
-            # '-q',  # be quiet
-            "-a",  # always write all output files
-            "-b", "htmlhelp",
-            "-E",  # Don’t use a saved environment (the structure
-                   # caching all cross-references),
-            "-N",  # Prevent colored output.
-            "-P",  # (Useful for debugging only.) Run the Python debugger,
-                   # pdb, if an unhandled exception occurs while building.
-            "-D", "release=%s" % self.buildSetup.appVersion,
-            "-D", "templates_path=[]",
-            "-d", EncodePath(join(self.buildSetup.tmpDir, ".doctree")),
-            '-v', '-v',  # verbosity, can be given up to three times
-            # write warnings and errors to file
-            # '-w', join('output', 'sphinx_log.txt'),
-            EncodePath(DOCS_SOURCE_DIR),
-            tmpDir,
-        ])
+        WritePluginList(join(self.buildSetup.docsDir, "pluginlist.rst"))
```

Comment:
the reason why i do not know if it is complete or not is I do not know if it is automatically generated. because there is a "pluginlist.rst" file. if it was something that was automatically generated would there be an rst file for the plugins? the docstrings are automatically generated for specific functions/methods in EG and those do not have an rst file.